rules/linux/credential_access_potential_linux_ssh_bruteforce.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2022/09/14"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies multiple consecutive login failures targeting an user account from the same source address and within a
+short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
+password, in an attempt to gain access to accounts.
```

Comment:
```suggestion
Identifies multiple consecutive SSH login failures targeting an user account from the same source address and within a
short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
password, in an attempt to gain access to accounts.
```

Reply:
The second part describes a password spraying attack, it would be cool if you adjust it to describe a brute force targeting a specific account.